queries.sql

```diff
@@ -150,3 +150,7 @@ WHERE ve.name = 'Maisy Smith'
 GROUP BY s.name
 ORDER BY num_visits DESC
 LIMIT 1;
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits WHERE vet_id = 2;
+EXPLAIN ANALYZE SELECT * FROM owners WHERE email = 'owner_18327@mail.com';
```

Comment:
- [ ] Before the performance audit, Please make sure all three queries have `Execution time: X ms and X >= 1000` currently, all three have Execution time <1000. I would suggest following the below steps to increase the execution time.

![image](https://github.com/aradradev/vet_clinic-database/assets/83803414/510b9766-6f80-4318-b69c-c100a5826d47)